mvlearn/embed/kcca.py

```diff
@@ -34,44 +59,75 @@ class KCCA(BaseEmbed):
     successful joint information reduction between two views
     and, therefore, routinely serves as a performance measure for CCA.
 
-    The kernel generalization of CCA, kernel CCA, is used when
-    there are nonlinear relations between two views.
+    CCA may not extract useful descriptors of the data because of
+    its linearity. kCCA offers an alternative solution by first
+    projecting the data onto a higher dimensional feature space.
 
-    Parameters
-    ----------
-    reg : float, default = 0.1
-          Regularization parameter
-    n_components : int, default = 10
-                   Number of canonical dimensions to keep
-    ktype : string, default = 'linear'
-            Type of kernel
-        - value can be 'linear', 'gaussian' or 'polynomial'
-    cutoff : float, default = 1x10^-15
-             Optional regularization parameter
-             to perform spectral cutoff when computing the canonical
-             weight pseudoinverse during held-out data prediction
-    sigma : float, default = 1.0
-            Parameter if Gaussian kernel
-    degree : integer, default = 2
-             Parameter if Polynomial kernel
+    .. math::
+        \phi: \mathbf{x} = (x_1,...,x_m) \mapsto
+        \phi(\mathbf{x}) = (\phi(x_1),...,\phi(x_N)),
+        (m < N)
+
+    before performing CCA in the new feature space.
+
+    Kernels are methods of implicitly mapping data into a higher
+    dimensional feature space, a method known as the kernel trick.
+    A kernel function K, such that for all :math:`\mathbf{x},
+    \mathbf{z} \in X`,
+
+    .. math::
+        K(\mathbf{x}, \mathbf{z}) = \langle\phi(\mathbf{x})
+        \cdot \phi(\mathbf{z})\rangle,
+
+    where :math:`\phi` is a mapping from X to feature space F.
 
+    The directions :math:`\mathbf{w_x}` and :math:`\mathbf{w_y}`
+    (of length N) can be rewritten as the projection of the data
+    onto the direction :math:`\alpha` and :math:`\alpha`
+    (of length m):
+
+    .. math::
+        \mathbf{w_x} = X'\alpha
+        \mathbf{w_y} = Y'\beta
+
+    Letting :math:`K_x = XX'` and :math:`K_x = XX'` be the kernel
+    matrices and adding a regularization term (:math:`\kappa`)
+    to prevent overfitting, we are effectively solving for:
+
+    .. math::
+        \rho = \underset{\alpha,\beta}{\text{max}}
+        \frac{\alpha'K_xK_y\beta}
+        {\sqrt{(\alpha'K_x^2\alpha+\kappa\alpha'K_x\alpha)
+        \cdot (\beta'K_y^2\beta + \kappa\beta'K_y\beta)}}
+
+
```

Comment:
Can you add an Examples section following the formats of the added sections in this PR: https://github.com/neurodata/mvlearn/pull/109/files

Reply:
yeah, i will work on this

Reply:
FYI, to make sure your Examples section passes, go to the mvlearn directory on your branch and do
```mvlearn
pytest --doctest-modules mvlearn/embed/kcca.py
```
if it passes, then it's good to go

Reply:
i get failures, whenever my examples try to print something.

ex. print("canonical correlations") --> expected nothing, got: "canonical correlations"

Reply:
looking at other examples. looks like i have to print it myself under.

Reply:
passed